packages/next/src/client/components/react-dev-overlay/internal/helpers/stitched-error.ts

```diff
@@ -26,15 +29,19 @@ export function getReactStitchedError<T = unknown>(err: T): Error | T {
   const newError = new Error(originMessage)
   // Copy all enumerable properties, e.g. digest
   Object.assign(newError, err)
-  newError.stack = newStack
 
   // Avoid duplicate overriding stack frames
   const ownerStack = (React as any).captureOwnerStack()
   if (ownerStack && newStack.endsWith(ownerStack) === false) {
     newStack += ownerStack
-    // Override stack
-    newError.stack = newStack
   }
 
+  if (locationInfo) {
+    newStack += `\n\n${locationInfo}`
```

Comment:
is `locationInfo` a stack? Are Browser Devtools still able to sourcemap both the owner stack and the location info?

Reply:
reasonable, I changed to format so that the stack is still not messed up